templates/base/src/components/atoms/Counter/Counter.js

```diff
@@ -0,0 +1,11 @@
+import { useState, useCallback } from 'react'
```

Comment:
I have created this dummy file to test react hook testing library, 
this file will be remove when someone write test case for react hook.